app/models/person.rb

```diff
@@ -56,6 +56,10 @@ def biography
     details["body"]
   end
 
+  def ordered_current_appointments
```

Comment:
I've [had to change the way the links are presented again](https://github.com/alphagov/collections/pull/1371#issuecomment-556978645), unfortunately!

When you rebase, you'll see there's a method you can use instead of this one though called `current_role_appointments`.